modules/platforms/dotnet/Apache.Ignite.Core.Tests/Services/ServicesTypeAutoResolveTest.cs

```diff
@@ -155,39 +157,59 @@ public void TestCallJavaServiceLocal()
         [Test]
         public void TestCallJavaServiceRemote()
         {
-            // Deploy Java service
-            var javaSvcName = TestUtils.DeployJavaService(_grid1);
+            DoTestPlatformService(name => _client.GetServices().GetServiceProxy<IJavaService>(name, false));
+        }
 
-            var svc = _client.GetServices().GetServiceProxy<IJavaService>(javaSvcName, false);
+        /// <summary>
+        /// Tests Java service invocation on remote node..
+        /// Types should be resolved implicitly.
+        /// </summary>
+        [Test]
+        public void TestCallJavaServiceThin()
+        {
+            DoTestPlatformService(name => _thinClient.GetServices().GetServiceProxy<IJavaService>(name));
+        }
 
-            DoTestService(svc);
+        /// <summary>
+        /// Tests .Net service invocation.
+        /// </summary>
+        public void DoTestPlatformService(Func<String, IJavaService> svc, bool isPlatform = false)
```

Comment:
The method name (and comment) is confusing, it can invoke java service as well as .net service, but according to method name and comment it can invoke only .net service.